pyodata/v2/model.py

```diff
@@ -361,6 +362,38 @@ def from_literal(self, value):
         return matches.group(1)
 
 
+def ms_since_epoch_to_datetime(value, tzinfo):
+    """Convert milliseconds since midnight 1.1.1970 to datetime"""
+    try:
+        # https://stackoverflow.com/questions/36179914/timestamp-out-of-range-for-platform-localtime-gmtime-function
+        return datetime.datetime(1970, 1, 1, tzinfo=tzinfo) + datetime.timedelta(milliseconds=int(value))
+    except (ValueError, OverflowError):
+        if FIX_SCREWED_UP_MINIMAL_DATETIME_VALUE and int(value) < -62135596800000:
```

Comment:
The -62135596800000 and 253402300799999 are obviously the "magic numbers", that repeats in if statement and the log error message as well. Local constant in the method would be nicer.

you  see the statement yourself:
` if FIX_SCREWED_UP_MAXIMUM_DATETIME_VALUE and int(value) > 253402300799999:`